server.ts

```diff
@@ -15,12 +17,31 @@ app
   .use(helmet())
   .use(morgan("dev"))
   .use(express.json())
-  .use(express.urlencoded({ extended: true }));
+  .use(express.urlencoded({ extended: true }))
+  .use(
+    cookieSession({
+      name: "session",
+      keys: [
+        process.env.COOKIE_SECRET
+          ? process.env.COOKIE_SECRET
+          : "supersecretkey",
```

Comment:
I think you can reduce this to process.env.COOKIE_SECRET ?? "supersecretkey" :)

Reply:
`||` since `process.env.*` returns `string | undefined` instead of `null` ^_^